src/Command/Docker/BuildIntegration.php

```diff
@@ -78,17 +82,27 @@ protected function configure()
         $this->setName(self::NAME)
             ->setDescription('Build test docker configuration')
             ->addArgument(
-                self::ARGUMENT_PHP,
+                self::ARGUMENT_MODE,
```

Comment:
Describe what mode is this. And possible values for it.

Reply:
Added

Reply:
Changed to `version`. Should be more clear now